docs/content/querying/sql.md

```diff
@@ -31,12 +31,12 @@ title: "SQL"
 
 # SQL
 
-<div class="note caution">
-Built-in SQL is an <a href="../development/experimental.html">experimental</a> feature. The API described here is
-subject to change.
+<div class="note info">
```

Comment:
Should the unsupported features of native Druid queries be called out a bit more prominently since we are moving this out of experimental status? Currently they are maybe sort of hidden after all of the functions it does support. Maybe it isn't a problem, but maybe a link from somewhere near the top to that section would be good?

Reply:
IMO, it’s decent the way it is — at least the unsupported features are called out on the same page.